utils/externalUrls.ts

```diff
@@ -146,3 +148,28 @@ export const universalSwapsAssetUrl = (address?: Address) => {
   assert(isAddress(address), 'Address is not a valid address')
   return `https://universalswaps.io/tokens/lukso/${address.toLowerCase()}`
 }
+
+/**
+ * URL to Mobile App deep link used by Wallet Connect
+ *
+ * @param data
+ * @returns
```

Comment:
I'd remove this if you are not planning to write anything here.

Reply:
Makes sense